bin/toolbox/presubmit/package_coverage_check.py

```diff
@@ -0,0 +1,15 @@
+from istio_toolbox.pkgCheck import pkg_cc_check
```

Comment:
Just out of curiosity: why not put this file in the toolbox repo? It seems to be repo-independent?